cumulusci.yml

```diff
@@ -166,3 +275,32 @@ orgs:
         prerelease:
             config_file: orgs/prerelease.json
 
+plans:
+    install:
+        slug: install
+        title: Install Volunteers for Salesforce
+        tier: primary
+        is_listed: False
+        steps:
+            1:
+                task: ensure_record_type
+                ui_options:
+                    name: Default Campaign Record Type
+            2:
+                task: install_managed
+                ui_options:
+                    name: Install Volunteers for Salesforce
+            3:
+                task: deploy_npsp_v4s_page_layouts
+                checks:
+                    - when: "'npsp' not in tasks.get_installed_packages()"
+                      action: hide
+                ui_options:
+                    is_required: False
+            3:
+                task: deploy_v4s_only_page_layouts
+                checks:
+                    - when: "'npsp' in tasks.get_installed_packages()"
+                      action: hide
+                ui_options:
+                    is_required: False
```

Comment:
I'm concerned that these tasks have potential to overwrite customizations to Contact Layout and Contact Lightning Layout, if V4S is installed in an org where the admin had already modified the layouts. Should we instead add a new layout called something like Volunteers for Salesforce Layout and provide instructions on how to switch to it if desired?

Reply:
@davisagli  Good call. I added a placeholder name and will discuss further with @scottwarren-sfdo and @aordoquiSFDO.